.github/workflows/release-drafter.yml

```
@@ -21,7 +21,17 @@ jobs:
           cache: gradle
       - name: Build with Gradle
         run: |
-          ./gradlew publishPublishMavenPublicationToLocalRepoRepository && tar -C build -cvf artifacts.tar.gz repository
+          ./gradlew publishPublishMavenPublicationToLocalRepoRepository
+      - name: Build shadowJar and include in artifacts
+        run: |
+          ./gradlew shadowJar
+          JDBC_VERSION=$(./gradlew :properties | grep version | cut -d ' ' -f 2)
+          SHADOW_JAR=$(echo build/libs/opensearch-sql-jdbc-shadow-"$JDBC_VERSION".jar)
+          ARTIFACTS=$(echo build/repository/org/opensearch/driver/opensearch-sql-jdbc/"$JDBC_VERSION")
+          mv $SHADOW_JAR $ARTIFACTS
```

You can simplify that
```suggestion
          mv build/libs/opensearch-sql-jdbc-shadow-*.jar build/repository/org/opensearch/driver/opensearch-sql-jdbc/*/
```
Just confirm (manually test) that corresponding directory is created.

---

Worked! Changed in https://github.com/Bit-Quill/sql-jdbc/pull/8/commits/f28d5adfdd5692ec74d177bb727a967f1abc4463